apps/src/studio/studio.js

```diff
@@ -1093,9 +1093,11 @@ Studio.init = function(config) {
   // pre-load images asynchronously
   // (to reduce the likelihood that there is a delay when images
   //  are changed at runtime)
-  preloadActorImages();
-  preloadProjectileImages();
-  preloadBackgroundImages();
+  if (config.skin.id == "studio") {
```

Comment:
Why do we not want to preload for infinity? Also, I'd rather see few to no checks for skin.id in studio.js. Instead, we should add a flag to the skin (i.e. skin.preloadAssets) and toggle our behavior in that.